R/describe_distribution.R

```diff
@@ -34,14 +38,79 @@
 #' data(iris)
 #' describe_distribution(iris)
 #' describe_distribution(iris, include_factors = TRUE, quartiles = TRUE)
+#' describe_distribution(list(mtcars$mpg, mtcars$cyl))
 #' @export
 describe_distribution <- function(x, ...) {
   UseMethod("describe_distribution")
 }
 
+#' @rdname describe_distribution
+#' @export
+
+describe_distribution.list <- function(x,
+                                       centrality = "mean",
+                                       dispersion = TRUE,
+                                       iqr = TRUE,
+                                       range = TRUE,
+                                       quartiles = FALSE,
+                                       ci = NULL,
+                                       include_factors = FALSE,
+                                       iterations = 100,
+                                       threshold = .1,
+                                       verbose = TRUE,
+                                       ...) {
+
+  # get elements names as is
+  # ex: list(mtcars$mpg, mtcars$cyl) -> c("mtcars$mpg", "mtcars$cyl")
+  nm <- sapply(sys.call()[[2]], deparse)[-1]
+
+  # Not possible to obtain elements names if they are stored in
+  # an object
+  if (length(nm) == 0) {
+    nm <- paste0("Var_", 1:length(x))
+  }
+
+  out <- do.call(rbind, lapply(x, function(i) {
+    if ((include_factors && is.factor(i)) || (!is.character(i) && !is.factor(i))) {
```

Comment:
For code readability, you should comment here the intent behind this complex conditional statement.

For example, 
> "The function currently doesn't support descriptive summaries for `character` or `factor` types."

Reply:
I copied this part from `describe_distribution.dataframe()`, so it should be modified there also. One question about that is that `describe_distribution()` also support characters and factors. I think the output is slightly different for characters and factors, which is why it can't be aggregated in a single dataframe with numeric variables, but it should be possible to output two different tables *à la* `skimr`. What do you think?

Reply:
Comments should suffice for now.

Yeah, we are also not that happy with its behaviour with these types, but this should be handled in different PR.

See #46 and #45